apps/email/js/sync.js

```diff
@@ -62,13 +62,11 @@ define(function(require) {
         //TODO: consider setting dir and lang?
         //https://developer.mozilla.org/en-US/docs/Web/API/notification
         var notificationOptions = {
-          body: body,
+          bodyL10n: body,
```

Comment:
if `body` is not an L10n Object, it should be turned into one as `{raw: body }`.